tests/event_store/subscriptions/test_handling_uncommitted_messages_from_other_clients.py

```diff
@@ -1,15 +1,64 @@
-import pytest
+from collections.abc import Iterator
+from pathlib import Path
 
-from event_sourcery.event_store import StreamId
+import pytest
+from django.db import transaction as django_transaction
+
+from event_sourcery.event_store import Backend, StreamId
+from event_sourcery_django import DjangoBackend
+from event_sourcery_sqlalchemy import SQLAlchemyBackend
+from tests import mark
+from tests.backend.django import django
+from tests.backend.sqlalchemy import (
+    sqlalchemy_postgres,
+    sqlalchemy_postgres_session,
+    sqlalchemy_sqlite,
+    sqlalchemy_sqlite_session,
+)
 from tests.bdd import Given, Then, When
 from tests.event_store.subscriptions.other_client import OtherClient
 from tests.factories import OtherEvent, an_event
 from tests.matchers import any_record
 
-pytestmark = pytest.mark.skip_backend(
-    backend=["kurrentdb", "in_memory"],
-    reason="Required only for SQL-based backends with transactions",
-)
+
+@pytest.fixture(params=[django, sqlalchemy_postgres, sqlalchemy_sqlite])
+def clients(
+    request: pytest.FixtureRequest,
+    tmp_path: Path,
+) -> Iterator[tuple[Backend, OtherClient]]:
+    backend_name: str = request.param.__name__
+    backend: Backend = request.getfixturevalue(backend_name)
+    mark.skip_backend(request, backend_name)
+
+    match backend_name:
```

Comment:
[trivia] IMO match when you're matching simple strings is an overkill and simple if-elif would suffice. Plus, there would be  fewer indentations by one level.